community/kernel/src/main/java/org/neo4j/unsafe/batchinsert/internal/BatchInserterImpl.java

```diff
@@ -278,7 +280,9 @@ public BatchInserterImpl( final File storeDir, final FileSystemAbstraction fileS
         indexStoreView = new NeoStoreIndexStoreView( LockService.NO_LOCK_SERVICE, neoStores );
 
         Dependencies deps = new Dependencies();
-        deps.satisfyDependencies( fileSystem, config, logService, indexStoreView, pageCache );
+        Monitors monitors = new Monitors();
+        Health databaseHealth = new Health.Adapter();
```

Comment:
We need _some_ `Health` to make the `KernelExtension` load and we don't need full `DatabaseHealth` so might as well use most simple one. No?